tests/jest/unit/geo/CentrelineUtils.spec.js

```diff
@@ -40,6 +41,28 @@ test('CentrelineUtils.getLocationFeatureType', () => {
   }).toThrow(InvalidCentrelineTypeError);
 });
 
+test('CentrelineUtils.getLocationStudyTypes', () => {
+  RoadIntersectionType.enumValues.forEach(({ featureCode }) => {
+    const location = {
+      centrelineType: CentrelineType.INTERSECTION,
+      featureCode,
+    };
+    expect(getLocationStudyTypes(location)).toBeInstanceOf(Array);
+  });
+  RoadSegmentType.enumValues.forEach(({ featureCode }) => {
+    const location = {
+      centrelineType: CentrelineType.SEGMENT,
+      featureCode,
+    };
+    expect(getLocationStudyTypes(location)).toBeInstanceOf(Array);
+  });
+  const location = {
+    centrelineType: CentrelineType.SEGMENT,
+    featureCode: null,
+  };
+  expect(getLocationStudyTypes(location)).toBeInstanceOf(Array);
```

Comment:
add a test that `MAJOR_ARTERIAL` includes `RESCU`, while `MINOR_ARTERIAL` does not